tgui/packages/tgui/interfaces/PreferencesMenu/jobs/jobs/research_director.ts

```diff
@@ -7,6 +7,7 @@ const ResearchDirector: Job = {
     order, make sure the AI and its Cyborgs aren't rogue, replacing them if \
     they are",
   department: Science,
+  alt_titles: ["Research Director", "Silicon Administrator", "Lead Researcher", "Biorobotics Director", "Research Supervisor"],
```

Comment:
```suggestion
  alt_titles: ["Research Director", "Silicon Administrator", "Lead Researcher", "Biorobotics Director", "Research Supervisor", "Chief Science Officer"],
```

cmon man my titles were good...